src/pid_ros.cpp

```diff
@@ -164,80 +164,80 @@ bool PidROS::getDoubleParam(const std::string & param_name, double & value)
 
-  getBooleanParam(param_prefix_ + "antiwindup", antiwindup);
+  get_boolean_param(param_prefix_ + "antiwindup", antiwindup);
 
   if (all_params_available) {
-    setParameterEventCallback();
+    set_parameter_event_callback();
   }
 
-  pid_.initPid(p, i, d, i_max, i_min, antiwindup);
+  pid_.initialize(p, i, d, i_max, i_min, antiwindup);
 
   return all_params_available;
 }
 
-void PidROS::declareParam(const std::string & param_name, rclcpp::ParameterValue param_value)
+void PidROS::declare_param(const std::string & param_name, rclcpp::ParameterValue param_value)
 {
   if (!node_params_->has_parameter(param_name)) {
     node_params_->declare_parameter(param_name, param_value);
   }
 }
 
-void PidROS::initPid(double p, double i, double d, double i_max, double i_min, bool antiwindup)
+void PidROS::initialize(double p, double i, double d, double i_max, double i_min, bool antiwindup)
 {
   if (i_min > i_max) {
     RCLCPP_ERROR(node_logging_->get_logger(), "received i_min > i_max, skip new gains");
   } else {
-    pid_.initPid(p, i, d, i_max, i_min, antiwindup);
+    pid_.initialize(p, i, d, i_max, i_min, antiwindup);
 
-    declareParam(param_prefix_ + "p", rclcpp::ParameterValue(p));
-    declareParam(param_prefix_ + "i", rclcpp::ParameterValue(i));
-    declareParam(param_prefix_ + "d", rclcpp::ParameterValue(d));
-    declareParam(param_prefix_ + "i_clamp_max", rclcpp::ParameterValue(i_max));
-    declareParam(param_prefix_ + "i_clamp_min", rclcpp::ParameterValue(i_min));
-    declareParam(param_prefix_ + "antiwindup", rclcpp::ParameterValue(antiwindup));
+    declare_param(param_prefix_ + "p", rclcpp::ParameterValue(p));
+    declare_param(param_prefix_ + "i", rclcpp::ParameterValue(i));
+    declare_param(param_prefix_ + "d", rclcpp::ParameterValue(d));
+    declare_param(param_prefix_ + "i_clamp_max", rclcpp::ParameterValue(i_max));
+    declare_param(param_prefix_ + "i_clamp_min", rclcpp::ParameterValue(i_min));
+    declare_param(param_prefix_ + "antiwindup", rclcpp::ParameterValue(antiwindup));
 
-    setParameterEventCallback();
+    set_parameter_event_callback();
   }
 }
 
 void PidROS::reset() { pid_.reset(); }
 
-std::shared_ptr<rclcpp::Publisher<control_msgs::msg::PidState>> PidROS::getPidStatePublisher()
+std::shared_ptr<rclcpp::Publisher<control_msgs::msg::PidState>> PidROS::get_pid_state_publisher()
 {
   return state_pub_;
 }
 
-double PidROS::computeCommand(double error, rclcpp::Duration dt)
+double PidROS::compute_command(double error, rclcpp::Duration dt)
 {
-  double cmd_ = pid_.computeCommand(error, static_cast<uint64_t>(dt.nanoseconds()));
-  publishPIDState(cmd_, error, dt);
+  double cmd = pid_.compute_command(error, dt);
+  publish_pid_state(cmd, error, dt);
 
-  return cmd_;
+  return cmd;
 }
 
-double PidROS::computeCommand(double error, double error_dot, rclcpp::Duration dt)
+double PidROS::compute_command(double error, double error_dot, rclcpp::Duration dt)
```

Comment:
I was thinking, whether the dt should be a const reference or a copy like this. I'm referring to here and rest of the code

Any opinion @christophfroehlich ?

Reply:
done

Reply:
For double, not needed
but it is fine